File: CS/WindowsFormsApplication18/MRUComboBoxEditPopupListBoxForm.cs
using DevExpress.XtraEditors;
using DevExpress.XtraEditors.Popup;

namespace MRUComboBox
{
	public class MRUComboBoxEditPopupListBoxForm : PopupListBoxForm
	{
		public MRUComboBoxEditPopupListBoxForm(ComboBoxEdit ownerEdit)
			: base(ownerEdit)
		{
		}

		protected override PopupListBox CreateListBox()
		{
			return new MRUComboBoxEditPopupListBox(this);
		}
	}
}

File: VB/WindowsFormsApplication18/MRUComboBoxEditPopupListBoxForm.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.XtraEditors
Imports DevExpress.XtraEditors.Popup

Namespace MRUComboBox
	Public Class MRUComboBoxEditPopupListBoxForm
		Inherits PopupListBoxForm
		Public Sub New(ByVal ownerEdit As ComboBoxEdit)
			MyBase.New(ownerEdit)
		End Sub

		Protected Overrides Function CreateListBox() As PopupListBox
			Return New MRUComboBoxEditPopupListBox(Me)
		End Function
	End Class
End Namespace
